### PR TITLE
Some work on html

### PR DESCRIPTION
**Pull request by fisx**

### CODE REVIEW

File: src/Frontend/Page/Idea.hs [hunk 1]
```diff
@@ -97,12 +107,12 @@ instance ToHtml ViewIdea where
                 idea ^. createdByLogin . fromUserLogin . html
                 " / "
                 when (phase `elem` [Nothing, Just PhaseRefinement]) $ do
-                    totalLikes ^. showed . html <> " Likes"  -- FIXME: singular
-                    " / "
+                    numberWithUnit totalLikes "Like" "Likes"
```
Comment (andorp) [inline src/Frontend/Page/Idea.hs]: Does the `"Likes"` is meaningful in German?

**Comment by andorp** [comment]
LGTM

File: src/Frontend/Page/Idea.hs (hunk 1 above)
Comment (fisx) [inline src/Frontend/Page/Idea.hs]: yes, i think we discussed that and decided it is the best "translation".

**Comment by np** [comment]
LGTM

**Pull request merged**